src/idsk/components/card/template.njk

```diff
@@ -1,36 +1,38 @@
 {% set type = params.type if params.type else "basic" %}
-{% set headingSize = "m" if type == 'simple' or type == 'secondary' else "l" %}
 
-<div class="idsk-card-{{ type }}">
+{% set dateAndTag %}
+    {% if params.date and params.tags %}
+        {% set tags = '<a  href="' + params.tags[0].link + '" class="idsk-tag-date idsk-tag-date-underline">' + params.tags[0].tag  + '</a>' %}
+        {% for index in range(1, params.tags | length) %}
```

Comment:
nestacilo by len {% for tag in tags %} ? ci preco to tu mame takto komplikovanejsie? :)